lib/sqlalchemy/orm/mapped_collection.py

```diff
@@ -22,9 +22,21 @@
 from ..sql import expression
 from ..sql import roles
 
+if TYPE_CHECKING:
+    from typing import List
+    from typing import Optional
+    from typing import Tuple
+    from typing import Union
+
+    from . import InstrumentedAttribute
+    from . import Mapper
+    from ..sql.elements import ColumnElement
+
 _KT = TypeVar("_KT", bound=Any)
 _VT = TypeVar("_VT", bound=Any)
 
+_AnyCallable = TypeVar("_AnyCallable", bound=Callable[[Any], Any])
```

Comment:
let's call this _F, which we've used in other files

Reply:
I have fixed this in latest commit, could you please take a look?